canonical/canonical.com
# Update search search bar and remove /careers/departments

## Done

- Removed /careers/departments page and added redirect
- Fixed width of search on /careers

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
   - https://canonical-com-725.demos.haus/careers
   - See that search bar styling has been correctly reverted for this page and is longer
   -  https://canonical-com-725.demos.haus/careers/all
   - See that the search bar here is still only takes up half the row
   - https://canonical-com-725.demos.haus/careers/departments
   - See that you're redirected to /careers  

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-933 https://warthogs.atlassian.net/browse/WD-934
